out/production/max_EDU/test/com/src/stream_api/StreamApi3.java

```diff
@@ -0,0 +1,25 @@
+package stream_api;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Створити List<String> numbers за допомогою методу List.of додати в нього назви цифр словами (one, two, three..) від 0 до 9.
+ * В наступній стрічці зробити стрім, знайти найдовше слово, використати .collect(Collectors.joining.. ) для отримання формату для прикладу “(t,w,o)”
+ */
+public class StreamApi3 {
+    public static void main(String[] args) {
+        List<String> list = List.of("zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine");
+
+        String result = list.stream()
+                .max(Comparator.comparing(String::length)).orElseThrow();
```

Comment:
orElseThrow(() -> new IllegalArgumentException()). Or what exception do you expect from orElseThrow() method here?

Reply:
orElseThrow(NoSuchElementException::new); --- for example I wanned finish stream with method which can return value or exception (not value)